ledger/src/blockstore.rs

```diff
@@ -2604,6 +2607,28 @@ impl Blockstore {
         self.perf_samples_cf.put(index, perf_sample)
     }
 
+    // program_cost table (if possible), it'd break a bit of pattern for ReplayStage to
+    // write. Can CF be gloabl - not tethed to slot?
```

Comment:
This has been answered, and can be removed now right?

Reply:
ah yes.